eureka-webapp/src/main/webapp/assets/js/eureka.registration.js

```diff
@@ -137,7 +137,14 @@ window.eureka.registration = new function () {
 				},
 				error: function (xhr, err) {
 					$('#passwordChangeFailure').show();
-					$('#passwordErrorMessage').text(xhr.responseText)
+                                        if (xhr.status == "409") {
```

Comment:
Please make this a switch statement. Also, status should be returning a numerical value not a string.